README.md

```diff
@@ -20,6 +20,7 @@
   - [Files](/API/files)
     - [`add`](/API/files#add)
     - [`createAddStream`](/files#createaddstream)
+    - [`get`](/API/files#get)
```

Comment:
@noffle seems that this was never added to the files.md. Maybe a missing commit?